src/components/app/details/cicdHistory/Constants.ts

```diff
@@ -19,4 +19,8 @@ export const HISTORY_LABEL = {
     APPLICATION: 'Application',
     ENVIRONMENT: 'Environment',
     PIPELINE: 'Pipeline',
-}
\ No newline at end of file
+}
+
+export const WORKER_POD_BASE_URL = '/resource-browser/1/devtron-ci/pod/k8sEmptyGroup'
+
+export const DEFAULT_TIMESTAMP = '0001-01-01T00:00:00Z'
```

Comment:
We already have `ZERO_TIME_STRING` constant. Please reuse that

Reply:
resolved